From 7b5d77b05d090d970a77ba007b15cd84f77abfe3 Mon Sep 17 00:00:00 2001
From: wkwkrnht <wkwkrnht@gmail.com>
Date: Tue, 4 Jul 2017 22:28:01 +0900
Subject: [PATCH] fix

---
 functions.php | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/functions.php b/functions.php
index 710aa43..0f1e061 100644
--- a/functions.php
+++ b/functions.php
@@ -18,7 +18,9 @@ function is_subpage(){
 }
 
 function is_actived_plugin($plugin = ''){
-    require_once(site_url('/includes/plugin.php'));
+    if(!is_admin()){
+        require_once(site_url('/includes/plugin.php'));
+    }
     return is_plugin_active($plugin);
 }
 
-- 
2.10.2.windows.1

